ansible/playbooks/sles.yml

```diff
@@ -65,24 +65,82 @@
         - ansible_distribution_major_version == "12"
       tags: build_tools
       
-    - name: Enable git-core repos on SLES 11
+    - name: Enable git-core repos - SLES 11 x86
       shell: |
         zypper addrepo 'http://download.opensuse.org/repositories/devel:/tools:/scm/SLE_11_SP4/devel:tools:scm.repo'
         zypper addrepo 'http://download.opensuse.org/repositories/devel:/languages:/perl/SLE_11_SP3/devel:languages:perl.repo'
         sed 's/gpgcheck=1/gpgcheck=0/' -i /etc/zypp/repos.d/devel_tools_scm.repo
         sed 's/gpgcheck=1/gpgcheck=0/' -i /etc/zypp/repos.d/devel_languages_perl.repo
       when:
         - ansible_distribution_major_version == "11"
+        - ansible_architecture == "x86"
       tags: build_tools
 
-    - name: Install git-core on SLES 11
+    - name: Install git-core - SLES 11 x86
       zypper: name={{item}} state=installed update_cache=yes
       with_items:
         - git-core
         - perl-Error
       when:
         - ansible_distribution_major_version == "11"
+        - ansible_architecture == "x86"
       tags: build_tools
+      
+    - name: Download Zlib - SLES 11 PPC64
```

Comment:
This might not be as likely for zlib, but maybe zlib and git should be in their own files and not specific to a platform.  For example on rhel7 the default git doesn't support options required and will likely need git rebuilt from source as well.

Reply:
Moving forward we need to start looking at if breaking out selections such as git is the best method for managing these playbooks. As the this playbooks are getting very long